app/views/support_interface/provider_users/edit.html.erb

```diff
@@ -12,6 +12,11 @@
       </li>
     </ol>
   </div>
+
+  <%= render SubNavigationComponent.new(items: [
+    { name: 'Details', url: support_interface_provider_user_path(@form.provider_user), current: true },
+    { name: 'Audit trail', url: support_interface_provider_user_audits_path(@form.provider_user) },
+  ]) %>
```

Comment:
Yes. It might be worth standardising the approach from https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/blob/52523e485b399dc05d60d3de09bd225b6933bc81/app/controllers/provider_interface/application_choices_controller.rb#L61-L79 or something like that?

Reply:
I now see that doesn't set `current: true` — maybe it can be improved at the same time

Reply:
Added a card for the tech debt backlog https://trello.com/c/T07i2UZF/238-can-we-extract-a-pattern-for-subnavigationcomponents